src/commands/slaveof.json

```diff
@@ -1,14 +1,20 @@
 {
     "SLAVEOF": {
-        "summary": "Make the server a replica of another instance, or promote it as master. Deprecated starting with Redis 5. Use REPLICAOF instead.",
+        "summary": "Make the server a replica of another instance, or promote it as master.",
         "complexity": "O(1)",
         "group": "server",
         "since": "1.0.0",
         "arity": 3,
         "function": "replicaofCommand",
+        "deprecated_since": "5.0.0",
+        "replaced_by": "`REPLICAOF`",
+        "doc_flags": [
+            "DEPRECATED"
```

Comment:
I'm not sure there's a reason to depreciate it.. The slave word is used everywhere. 
Until we find a solution for info and other places, there's no reason to proceed in that direction IMHO.

Reply:
ohh, i did see the summary said. i am ok with no touching it, it's not a problem
> "summary": "Make the server a replica of another instance, or promote it as master. Deprecated starting with Redis 5. Use REPLICAOF instead.",

Reply:
i see the text was changed in https://github.com/redis/redis-doc/commit/dabc29ce (not mentioning "deprecated")
and it looks like the deprecation word was added only when creating the slaveof.json file.
@itamarhaber please share your thoughts.

Reply:
I do see the point in marking this as deprecated. While the s-word isn't going away soon, the actual command has an alternative and we should direct people to use it.

Reply:
ok, just let's consider the implications..  we're not talking about some user facing text, but also a flag.
some clients may decide to issue a warning when this command is used.
i guess they won't.. since there are a ton of other heavily used commands already marked as deprecated.
so @enjoy-binbin feel free to revert your revert.

Reply:
ok, added back, and update the top comment and the title